src/suricata.c

```diff
@@ -3000,6 +3000,14 @@ int InitGlobal(void)
 
 void SuricataPreInit(const char *progname)
 {
+    /*
```

Comment:
Would like to see this be a bit more compact, like in a helper func that is just called from here